app/rtc_s1.py

```diff
@@ -1273,4 +1288,4 @@ def get_rtc_s1_parser():
     _load_parameters(cfg)
 
     # Run geocode burst workflow
-    run(cfg)
+    run(cfg)
```

Comment:
I think it's better with a new line at the end?

Reply:
Yes, it would be better with the new line at the end. Thanks for catching this.